.idea/misc.xml

```diff
@@ -1,4 +1,2075 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for?

Reply:
Sets up entry points in IDEA, so that classes/methods are not reported as unused.